### ISSUE TITLE
Create tests!

### ISSUE THREAD

**Issue by sdorkenw** [knossos-project/knossos_utils#16]
We desperately need tests! Things get changed without testing their implications and effects on the functionality...

**Comment by mdraw** [comment]
Yes, something like a test suite with http://pytest.org and eventually automated testing using tox and CI services will be a huge improvement and make collaboration easier. Does anyone have concrete plans for this?
